elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -178,6 +177,8 @@ async def _verify_attached_flaws_for(self, advisory_id: int, attached_trackers:
         # Retrieve flaw bugs for attached_tracker_bugs
         tracker_flaws, flaw_id_bugs = BugTracker.get_corresponding_flaw_bugs(attached_trackers,
                                                                              self.runtime.bug_trackers('bugzilla'))
+        current_cve_package_exclusions = await AsyncErrataUtils.get_advisory_cve_package_exclusions(self.errata_api, advisory_id)
```

Comment:
start the async call early to get the necessary content before is_first_fix_any function call

Reply:
even is_first_fix_any get bugs in batch will also take 8 mins for RC